python/src/ignition/gazebo/helper_system.hh

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef IGNITION_GAZEBO_PYTHON__HELPER_SYSTEM_HPP_
+#define IGNITION_GAZEBO_PYTHON__HELPER_SYSTEM_HPP_
+
+#include <pybind11/pybind11.h>
+#include <pybind11/functional.h>
+#include <pybind11/stl.h>
+
+#include <memory>
+#include <string>
+#include <functional>
+
+#include "ignition/gazebo/EntityComponentManager.hh"
+#include "ignition/gazebo/Export.hh"
+#include "ignition/gazebo/Server.hh"
+#include "ignition/gazebo/ServerConfig.hh"
+
+#include "destroyable.hh"
+#include "entity_component_manager.hh"
+
+namespace py = pybind11;
+
+namespace ignition
+{
+namespace gazebo
+{
+namespace python
+{
+  class HelperFixturePrivate;
+
+  class HelperFixture : public ignition::utils::python::Destroyable, public std::enable_shared_from_this<HelperFixture>
```

Comment:
Same here, this looks very similar to `TestFixture`; can we just use that class instead?

Reply:
Same comment https://github.com/ignitionrobotics/ign-gazebo/pull/1219#discussion_r756703333